onnxruntime/test/framework/kernel_type_str_resolver_utils_test.cc

```diff
@@ -26,8 +26,8 @@ static Status LoadLayoutTransformationRequiredOpsFromOpSchemas(KernelTypeStrReso
   return Status::OK();
 }
 
-TEST(KernelTypeStrResolverUtilsTest, DISABLED_VerifyLayoutTransformationRequiredOpsResolver) {  // actual_resolver.GetOpKernelTypeStrMap()
-                                                                                                //     Which is: { (com.microsoft:QLinearConv:1, { ("y_scale",
+TEST(KernelTypeStrResolverUtilsTest, VerifyLayoutTransformationRequiredOpsResolver) {  // actual_resolver.GetOpKernelTypeStrMap()
```

Comment:
thanks for enabling this.

the existing comment doesn't quite make sense, maybe we can remove it?